### ISSUE TITLE
The build fails in Travis

### ISSUE THREAD

**Issue by lfryc** [richwidgets/richwidgets#34]
https://travis-ci.org/richwidgets/richwidgets/builds/12246680

It seems bower deps are not downloaded correctly

**Comment by bleathem** [comment]
Note the travis reported error message is incorrect.  The call to _grunt bower_ fails with a message, but the message is suppressed do the the _grunt-shell_ invocation of bower install.  You can see the bower error by expanding the ">" next to the _grunt bower_ invocation in the travis log.  The suppressed error is:

> There were errors, here's a summary of them:
> - dom-compare Tag requirejs is not a valid semver range/version
> 
> Done, without errors.

I've replaced the _grunt bower_ invocation with _bower install_ in the travis conig file to get better error reporting in the event of _bower install_ failures.

**Comment by bleathem** [comment]
Note _package.json_ calls for bower version 0.9.2.  Running bower locally I found I was running bower version 1.1.0.  I updated package.json to use bower version 1.2.7, and updated my local install.  

We'll have to be sure to keep our local globally installed versions of bower in synch with that defined in package.json.  At least travis will let us know when the versions diverge enough to introduce a problem as in this issue.
